nb/api/Triangle.md

```diff
@@ -0,0 +1,30 @@
+### Triangle()
+Parameter|Default|Type
+---|---|---
+...dimensions|[1, 1, 1]|The size of the bounding box of the triangle.
+
+Produces an triangle or triangular prism that fits into the bounding box.
+
+```JavaScript
+Triangle(5).view().note('Triangle(5)');
+```
+
+![Image](Triangle.md.0.png)
+
+Triangle(5)
+
+```JavaScript
+Triangle(2, 4).view().note('Triangle(2, 4)');
+```
+
+![Image](Triangle.md.1.png)
+
```

Comment:
Something else that is nice about splitting these out into files is that the generated .png views follow the changed files more closely in the diffs. IE the diff of the file is located close to the diff of the generated .png when they are sorted alphabetically

Reply:
Yes, it involves more work, but it gives a better result.

The regression stuff is mostly just sticking an example in somewhere so that it breaks visibly next time.